frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -97,6 +98,13 @@ def send_notification_email(doc):
 		now=frappe.flags.in_test
 	)
 
+def get_email_body_content(doc):
+	#include document name in email content
+	title_html = get_title_html(get_title(doc.document_type, doc.document_name))
+	subject_parts = doc.subject.partition(title_html)
+	body_content = "{0}{1} ({2}){3}".format(subject_parts[0], subject_parts[1], doc.document_name, subject_parts[2])
+	return body_content
```

Comment:
Code looks hacky... can't we directly set document name to the subject?

Also, bit un-related, but
<img width="311" alt="Screenshot 2020-05-25 at 10 40 50 PM" src="https://user-images.githubusercontent.com/13928957/82832642-5044cf00-9ed9-11ea-8258-a80be805f2a7.png">
can we rename this to **Open Document** instead...